CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+We welcome contributions to our code, and will always endeavor to respond in a quick and helpful manner to Pull Requests.
+
+1. **Please sign one of the contributor license agreements below** (simple and pain-free).
+1. Fork the repo, develop and test your code changes, add docs.
```

Comment:
>2.  A patch should be a set of changes that addresses **one** issue in the issue tracker. Please file one change per issue, and address one issue per change. If you want to make a change that doesn't have a corresponding issue in the issue tracker, please file a new ticket!

Shamelessly ripped from one of the other client libraries contribute page, because it sensible and makes things easier to track.